config/config.go

```diff
@@ -294,6 +305,127 @@ func (m *ManualWarpMessage) Validate() error {
 	return nil
 }
 
+// If the numerator in the Warp config is 0, use the default value
+func setQuorumNumerator(cfgNumerator uint64) uint64 {
+	if cfgNumerator == 0 {
+		return warp.WarpDefaultQuorumNumerator
+	}
+	return cfgNumerator
+}
+
+// Helper to retrieve the Warp Quorum from the chain config.
+// Differentiates between subnet-evm and coreth RPC internally
+func getWarpQuorum(
+	blockchainID ids.ID,
+	subnetID ids.ID,
```

Comment:
Nit: Can we put `subnetID` as the first param here since it's technically a broader scope than `blockchainID`? 

Every blockchain ID belongs to exactly one subnet ID, but agree it's not worth doing the lookup of the subnet ID based on the blockchain ID here since it's easier to be passed both.